### ISSUE TITLE
Produce likelihood for each eye blink predictions

### ISSUE THREAD

**Issue by uetchy** [prism-3d/eyeblink#1]
Because we want to know if webcam actually grasps an image of eyes, the prediction likelihood needs to be added on the output layer.

# Refs

- [Eye Tracking Update: Neural Network Architecture and Loss Functions](https://medium.com/@rcdilorenzo/eye-tracking-update-neural-network-architecture-and-loss-functions-7392a3f5a2af)
